met-api/src/met_api/models/submission.py

```diff
@@ -81,3 +86,39 @@ def get_engaged_users(cls, engagement_id) -> List[User]:
             .filter(Survey.engagement_id == engagement_id)\
             .all()
         return users
+
+    @classmethod
+    def get_by_survey_id_paginated(cls, survey_id, pagination_options: PaginationOptions, search_text=''):
```

Comment:
Just thinking out loud. What made us need to query the submission table for this purpose. Why not query the comments table and  group by submission id?